src/Commands/BuildCommand.php

```diff
@@ -107,6 +126,10 @@ protected function buildHtml(string $path)
     {
         $this->output->writeln('<fg=yellow>==></> Parsing Markdown ...');
 
+        if (is_callable($this->config['prehtml'] ?? null)) {
```

Comment:
Invoking this output call here so it only renders once in the console as opposed to per file.

Might be nice to render this message per file along with the active filename so users can see which file might be breaking for them?